pkg/query-service/rules/apiParams.go

```diff
@@ -105,7 +110,7 @@ func parsePostableRule(content []byte, kind string) (*PostableRule, []error) {
 	if errs := rule.Validate(); len(errs) > 0 {
 		return nil, errs
 	}
-	return &rule, []error{}
+	return rule, []error{}
```

Comment:
redundant?
```suggestion
	return []error{}
```

Reply:
Fixed this. The input rule is passed by value, so the caller can retain old copy, and the return pointer will be for new value.